config/sync/pathauto.pattern.terms_default.yml

```diff
@@ -10,25 +10,19 @@ label: 'Terms - default'
 type: 'canonical_entities:taxonomy_term'
 pattern: '[term:vocabulary]/[term:name]'
 selection_criteria:
-  284e394b-6396-4f23-97a9-ad58d17847dd:
+  2743a7ed-18f1-4803-8bb9-5e9769ac1de3:
     id: 'entity_bundle:taxonomy_term'
     bundles:
-      accessni_ub_sectors: accessni_ub_sectors
-      accessni_ub_services: accessni_ub_services
-      districts_in_northern_ireland: districts_in_northern_ireland
       drive_instr_categories: drive_instr_categories
       hc_body_location: hc_body_location
       hc_body_system: hc_body_system
       hc_condition_type: hc_condition_type
       hc_info_sources: hc_info_sources
       hc_symptoms: hc_symptoms
-      recipe_course_types: recipe_course_types
```

Comment:
Not strictly needed but figured because you had it removed from the other task + then ran config export it'll also be removed here. They're both coming out anyway so no big deal